.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.3.9/apache-maven-3.3.9-bin.zip
```

Comment:
You are using older wrapper version

Reply:
New one can be pinned by `mvn -N io.takari:maven:wrapper -Dmaven=3.5.3`

Reply:
Fixed